package.json

```diff
@@ -26,7 +26,7 @@
     "start": "run-p start:**",
     "start:app": "react-scripts start",
     "start:lambda": "netlify-lambda serve src/backend/functions",
-    "build": "run-p build:**",
+    "build": "npm ci && run-p build:**",
```

Comment:
You might try replacing your package.json with the one from main because you ideally shouldn't have any changes